.sbtrc

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright 2021 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Aliases in this file expected usage of test file naming conventions:
+#   - "UnitTests" suffix for test files and Suites which define unit tests
+#   - "IntegrationTests" suffix for test files and Suites which define integration tests
+
+# CPS QA types aliases
+# * Unit tests
+alias test=; testOnly *UnitTests
```

Comment:
Problem is, that tasks that include testing like `assembly` don't take aliases into account and run them according to original definition. This creates an IMHO inconsistent behavior.

Reply:
This problem can be solved by adding this line into module settings in build.sbt.
`      (assembly / test) := {},`

Is that acceptable for modules with Integration tests and external dependencies?

Reply:
See the commit where with the proposed solution - ed76338048d7cfff5fb9e0f94681b06ea75fc41c